### PR TITLE
show open ports

### PR DESCRIPTION
**Pull request by dimmyjr**

### CODE REVIEW

**Comment by Darateja** [comment]
Hie sir I need a job

File: src/_renderer.js [hunk 1]
```diff
@@ -427,8 +427,9 @@ async function initUI() {
 
     // Right column
     window.mods.netstat = new Netstat("mod_column_right");
-    window.mods.globe = new LocationGlobe("mod_column_right");
+    // window.mods.globe = new LocationGlobe("mod_column_right");
```
Comment (GitSquared) [inline src/_renderer.js:430]: You can't deactivate existing modules, this is a merge blocker.

File: src/_renderer.js [hunk 2]
```diff
@@ -427,8 +427,9 @@ async function initUI() {
...
     window.mods.conninfo = new Conninfo("mod_column_right");
+    window.mods.openportlist = new OpenPortList("mod_column_right");
```
Comment (GitSquared) [inline src/_renderer.js:432]: Could you give me a screenshot/example of how this looks?

**Review by GitSquared — CHANGES_REQUESTED** [review]

**Comment by GitSquared** [comment]
closed as stale

**Comment by dereksmartymans** [comment]
Thanks for sending!   
  
  
  
  
  
    
  

  
  
>

**Comment by dereksmartymans** [comment]
Thank you, I appreciate it.   
  
  
  
  
  
    
  

  
  
>

**Pull request closed without merging**